h2o-algos/src/main/java/hex/tree/drf/DRFModel.java

```diff
@@ -19,8 +19,6 @@
     public DRFParameters() {
       super();
       // Set DRF-specific defaults (can differ from SharedTreeModel's defaults)
-      _mtries = -1;
```

Comment:
mtries default value is -1 defined at https://github.com/h2oai/h2o-3/pull/3843/files#diff-9ef751f7b3c63372a36f999ae67c0bd9L17. This is not necessary